Dockerfile-alpine.template

```diff
@@ -0,0 +1,103 @@
+FROM ruby:%%RUBY_VERSION%%-alpine3.10
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -S redmine && adduser -S -H -G redmine redmine
```

Comment:
Should we specify user and group IDs to ensure easy moving from one os variant to another? (i.e. use whatever the current IDs debian has and make them static for both)

Reply:
Unfortunately we can only line up the user id. :cry:
```console
$ docker run --rm --user redmine redmine id
uid=999(redmine) gid=999(redmine) groups=999(redmine)
$ docker run --rm alpine:3.10 sh -c ' cat /etc/passwd | grep 999'
$ docker run --rm alpine:3.10 sh -c ' cat /etc/group | grep 999'
ping:x:999:
```

And add redmine to the `ping` group? :confused: :nauseated_face: